example/centos/centos-9-aarch64-minimal-raw.yaml

```diff
@@ -0,0 +1,158 @@
+          options:
+            prefix: ''
+        - type: org.osbuild.locale
+          options:
+            language: C.UTF-8
+        - type: org.osbuild.timezone
+          options:
+            zone: America/New_York
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
+              update_default: true
+              default_kernel: kernel
+            network:
+              networking: true
+              no_zero_conf: true
+        - type: org.osbuild.systemd.unit
+          options:
+            unit: grub-boot-success.timer
+            dropin: 10-disable-if-greenboot.conf
+            config:
+              Unit:
+                ConditionPathExists: '!/usr/libexec/greenboot/greenboot'
+            unit-type: global
+        - otk.external.otk-make-fstab-stage:
+            ${filesystem}
+        - otk.include: fragment/grub2/aarch64.yaml
+        - type: org.osbuild.copy
+          inputs:
+            file-cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9:
+              type: org.osbuild.files
+              origin: org.osbuild.source
+              references:
+                - id: sha256:cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9
+          options:
+            paths:
+              - from: input://file-cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9/sha256:cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9
+                to: tree:///root/anaconda-ks.cfg
+                remove_destination: true
+        - type: org.osbuild.chown
+          options:
+            items:
+              /root/anaconda-ks.cfg:
+                user: root
+                group: root
+        - type: org.osbuild.systemd
+          options:
+            enabled_services:
+              - NetworkManager.service
+              - firewalld.service
+              - sshd.service
+              - initial-setup.service
+        - type: org.osbuild.selinux
+          options:
+            file_contexts: etc/selinux/targeted/contexts/files/file_contexts
+    - otk.include: pipeline/image/aarch64.yaml
+    - name: xz
+      build: name:build
+      stages:
+        - type: org.osbuild.xz
+          inputs:
+            file:
+              type: org.osbuild.files
+              origin: org.osbuild.pipeline
+              references:
+                name:image:
+                  file: disk.img
+          options:
+            filename: disk.raw.xz
+  sources:
+    otk.op.join:
+      values:
+        - otk.external.osbuild-make-depsolve-dnf4-curl-source:
+            packagesets:
+              - ${packages.build}
+              - ${packages.os}
+        - org.osbuild.inline:
+            items:
+              sha256:cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9:
+                encoding: base64
+                data: IyBSdW4gaW5pdGlhbC1zZXR1cCBvbiBmaXJzdCBib290CiMgQ3JlYXRlZCBieSBvc2J1aWxkCmZpcnN0Ym9vdCAtLXJlY29uZmlnCmxhbmcgZW5fVVMuVVRGLTgK
```

Comment:
I think we should not merge this before we have a way to represent this, i.e. let's use this example to bring back the external for inline data.

Reply:
#238

Reply:
I've updated this PR to use the external from #238.